app.py

```diff
@@ -1,37 +1,42 @@
-from flask import Flask, request, jsonify
+# from flask_sqlalchemy   import SQLAlchemy
+# from flask_marshmallow import Marshmallow
+# import code
+# code.interact(local=dict(globals(), **locals()))
```

Comment:
We might consider pulling these comments here out into a google doc

Reply:
Done!